news/2018-06-22-cwc-2018-recap-round-of-16.md

```diff
@@ -0,0 +1,59 @@
+
+Hong Kong then overcame a very game Norway, who took the bold decision to fight strength-for-strength, picking Hidden twice in succession and coming out the victors on both occasions. It was so nearly a third when Norway slid to a 6,500 point defeat on the low Approach Rate Hidden pick, [96Neko x KurousaP - Kagen no Tsuki](https://osu.ppy.sh/beatmapsets/393343#fruits/856125), but after the Hidden pool had been exhausted, Hong Kong's better bench strength and flexibility was brought to bear, storming to a 5-2 victory. The true plaudits in this match went to Groex of the Norwegian team however, as he fought his way to 5 Full Combo performances (and only four misses in the entire match) to ensure his team was always within striking distance throughout.
+
+The spotlight would continue to shine on Scandinavia next, as Finland would almost shock their hugely experienced Taiwanese counterparts in securing several narrow victories on the way to a delicately balanced first six games. Taiwan drew out the best from the Finnish team here, as they did in their Group Stage tiebreaker against France last week, but kept their cool under mounting pressure from the underdogs to complete a much narrower than expected 5-3 victory.
+
+Poland were forced to relive the horrors of 2017 as they were once again drawn against the ever-impressive China, but nonetheless fought valiantly to secure a game victory on their way to a 5-1 defeat. It was somewhat of an unfair scoreline for the Polish team, who held the Chinese juggernauts to a number of incredibly close games. However, as so often is the case when competing against such vastly experienced players, whenever the slightest window of opportunity was opened, it was immediately and ruthlessly seized upon.
+
+Another team who were left wondering “what if?” after their match were The Netherlands, whose strategy unraveled spectacularly after only a single game. Attempting to sneak an opening victory on the uncomfortable NoMod pick, [sana - Packet Hero](https://osu.ppy.sh/beatmapsets/694901#fruits/1469957), the Dutch team succumbed to a 15,500 point loss as the Germans held their nerve in a frantic back-and-forth exchange which had everyone out of their seats.
+
+![](/wiki/shared/news/2018-06-22-cwc-2018-recap-round-of-16/GERvNL_game1_720w.jpg)
+
+From this point The Netherlands confidence was shot, and, buoyed by their opening break point victory, Germany reeled off four further game wins in succession to complete the 5-0 sweep. This was some measure of revenge for Germany, who had been soundly beaten by the Dutch in last year’s Semifinals, and they played with neither fear nor mercy in a frightening display of consistency that their future opponents have surely taken notice of.
```

Comment:
uncapitalize "The" in the first sentence

Reply:
Fixed and also added comma